MBBSEmu/Module/MsgFile.cs

```diff
@@ -108,6 +108,9 @@ private void BuildMCV()
                     case MsgParseState.IDENTIFIER when IsIdentifier(c):
                         identifier.Append(c);
                         break;
+                    case MsgParseState.IDENTIFIER when c == '\r':
```

Comment:
`when c == '\r' || c == '\n'` would be better here I think, in case the MSG is formatted in Unix format without `\r`

Though I don't think this parser would succeed without `\r` anyhow, there are other places that look suspect